modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/register/impl/RegisterServiceImpl.java

```diff
@@ -130,9 +130,12 @@ private RegisterServiceEntity getNewRegisterServiceEntity(Ocl ocl) {
     @Override
     public UUID registerService(Ocl ocl) {
         RegisterServiceEntity newEntity = getNewRegisterServiceEntity(ocl);
-        if (Objects.nonNull(storage.findRegisteredService(newEntity))) {
-            log.error("Service already registered.");
-            throw new IllegalArgumentException("Service already registered.");
+        if (!newEntity.getOcl().getIcon().startsWith("data:image/png;base64,")) {
```

Comment:
I think it is not so good to assert the Icon here.

Reply:
yes. We must allow images with base 64 data.. and additionally also with URL.

Reply:
ok